Lib/idlelib/idle_test/test_tooltip.py

```diff
@@ -1,27 +1,31 @@
+from functools import wraps
```

Comment:
This first few changes are irrelevant to the issue and are a regression from the 'fail fast (as soon as possible)' design used in IDLE tests.  If importing the tested module fails, doing anything else is useless.  If the whole module requires gui and that fails (as is does for a majority of buildbots) doing anything else is useless.  I have changed back to this.

I am also adding module docstring with coverage.